adafruit_avrprog.py

```diff
@@ -229,7 +237,7 @@ def verify_file(self, chip, file_name, verbose=False):
         self.end()
         return True
 
-    def read_fuses(self, chip):
+    def read_fuses(self, chip: dict) -> list:
```

Comment:
It looks like this returns a tuple, which can also be annotated as a generic!  In the case of tuples, the generic can specify the types on the tuple's contents.  Since the tuple returned seems to be of length 4, all of which are `int`, we can import `Tuple` from `typing` and annotate it as:

```suggestion
    def read_fuses(self, chip: Dict[str, Any]) -> Tuple[int, int, int, int]:
```